src/hooks/useSettingsStatus.ts

```diff
@@ -0,0 +1,74 @@
+import {useOnyx} from 'react-native-onyx';
+import type {ValueOf} from 'type-fest';
+import {isConnectionInProgress} from '@libs/actions/connections';
+import * as PolicyUtils from '@libs/PolicyUtils';
+import * as SubscriptionUtils from '@libs/SubscriptionUtils';
+import * as UserUtils from '@libs/UserUtils';
+import * as PaymentMethods from '@userActions/PaymentMethods';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import useTheme from './useTheme';
+
+type SettingsStatus = {
+    indicatorColor: string;
+    status: ValueOf<typeof CONST.SETTINGS_STATUS> | undefined;
+    idOfPolicyWithErrors: string | undefined;
+};
+
+function useSettingsStatus(): SettingsStatus {
```

Comment:
```suggestion
type IndicatorStatus = {
    indicatorColor: string;
    status: ValueOf<typeof CONST.SETTINGS_STATUS> | undefined;
    idOfPolicyWithErrors: string | undefined;
};

function useIndicatorStatus(): IndicatorStatus {
```

Correct if I'm wrong, but shouldn't this be called `useIndicatorStatus` since its main purpose is to be used in the `Indicator` component?

If yes I think we should also rename `SETTINGS_STATUS` to `INDICATOR_STATUS`